hubspot/src/main/kotlin/org/boomevents/hubspot/domain/contact/ContactClient.kt

```diff
@@ -0,0 +1,87 @@
+package main.kotlin.org.boomevents.hubspot.domain.contact
+
+import main.kotlin.org.boomevents.hubspot.domain.contact.exception.ContactNotFoundException
+import org.boomevents.hubspot.Client
+import org.boomevents.hubspot.ClientRequestCatalog
+import org.boomevents.hubspot.domain.company.exceptions.CompanyNotFoundException
+import org.boomevents.hubspot.model.http.RequestMethod
+import org.boomevents.hubspot.model.http.Requester
+import org.boomevents.hubspot.model.http.exceptions.HttpRequestException
+import org.boomevents.hubspot.model.mapper.Mapper
+import java.math.BigInteger
+import java.util.Collections.emptyMap
+
+class ContactClient(private val hubSpotClient: Client) {
+
+    fun <P> createContact(request: ContactRequest<P>): Contact {
+        val response = Requester.requestJson(
+            hubSpotClient,
+            RequestMethod.POST,
+            ClientRequestCatalog.V3.CONTACTS,
+            emptyMap(),
+            request
+        )
+
+        if (response.isSuccess) {
+            return Mapper.mapToObject(response.body)
+        } else {
+            throw RuntimeException(response.statusText)
+        }
+    }
+
+    @Throws(
+        CompanyNotFoundException::class,
```

Comment:
Can you please change `@Throws` annotation from  `CompanyNotFoundException` to `ContactNotFoundException`?